bdd/features/e2e/E2E-003-kill.feature

```diff
@@ -7,7 +7,6 @@ Feature: Kill e2e tests
         And instance started
         And wait for instance healthy is "true"
         And get runner PID
-        And send kill message to instance
```

Comment:
Why is this removed? How are we testing if the sequence responds to kill?

Reply:
need to look into that and see why i removed it xd good catch :wink:

Reply:
ok, brought it back, It works. must have been accidental when I was improving some tests to allow the sequence to exit on it's own, without killing it.